src/DetailsView/components/failure-instance-panel-control.tsx

```diff
@@ -45,11 +49,24 @@ export class FailureInstancePanelControl extends React.Component<FailureInstance
 
     constructor(props) {
         super(props);
+        let filledOriginalInstance;
+        if (this.props.originalInstance) {
+            filledOriginalInstance = {
+                failureDescription: this.props.originalInstance.failureDescription,
+                path: this.props.originalInstance.path,
+                snippet: this.props.originalInstance.snippet,
+            };
+        } else {
+            filledOriginalInstance = { failureDescription: '', path: '', snippet: '' };
+        }
+
         this.state = {
             isPanelOpen: false,
-            failureDescription: this.props.originalText || '',
-            path: '',
-            snippet: '',
+            currentInstance: {
```

Comment:
This doesn't seem necessary; we can just use the instance you have from line 52.

Reply:
I did it this way because currently we could have had a valid Original Instance which had a comment, but didn't have a path or snippet. If we don't check the validity of each field I've run into errors where a field is null instead of ''. Do you think it's better to do this check during the creation of currentInstance above? Just let me know!